spec/views/shared/eligible_region_content_spec.rb

```diff
@@ -113,7 +112,7 @@
     before { FeatureFlags::FeatureFlag.activate(:application_work_history) }
     after { FeatureFlags::FeatureFlag.deactivate(:application_work_history) }
 
-    let(:region) { create(:region) }
+    let(:region) { create(:region, reduced_evidence_accepted: false) }
```

Comment:
I think we should add some tests for when `reduced_evidence_accepted: true` and check what content is being displayed to the user.